.github/workflows/test.yml

```diff
@@ -31,3 +31,5 @@ jobs:
         run: npm ci
 
       - run: npm run build
+
+      - run: git diff --exit-code
```

Comment:
@domoritz we might need to tweak the `created` timestamp generation on PackageMeta`.

IIRC it will update every time the script runs

Reply:
😂 https://github.com/vega/vega-datasets/pull/631/commits/2a7726b4df57d6ac7d7a321771fd91ed81136f7e